l10n_es_intrastat_report/security/l10n_es_intrastat_report_security.xml

```diff
@@ -6,6 +6,6 @@
         <field name="global" eval="True" />
         <field
             name="domain_force"
-        >[('company_id', 'child_of', user.company_id.ids)]</field>
+        >[('company_id', 'child_of', user.company_ids.ids)]</field>
```

Comment:
Although it's work fine i don't really sure that it's totally correct.

Reply:
This should be updated in v13:

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

Reply:
Yes, I missed that in the migration. Making the change now and then I'll rebase here.

Reply:
These changes it's merged in 13.0, please rebase.